### PR TITLE
[FSDE-6040] Fixed App layout web component title issue

### PR DESCRIPTION
**Pull request by janibolkvadze**
@SergioQCL 

https://ctl.atlassian.net/browse/FSDE-6040

### CODE REVIEW

File: src/custom-elements/src/components/app-layout/app-layout.tsx [hunk 1]
```diff
@@ -51,27 +51,6 @@ export class AppLayout {
...
-      };
-
-      this.mutationObserver = new MutationObserver(subscriberCallback);
-    }
-
-    this.mutationObserver.observe(observerTarget, mutationObserverConfig);
-  }
-
   disconnectedCallback() {
     this.mutationObserver.disconnect();
```
Comment (dani-cl-madrid) [inline src/custom-elements/src/components/app-layout/app-layout.tsx:55]: You might want to remove this lines as well.

File: src/custom-elements/src/components/app-layout/app-layout.tsx [hunk 2]
```diff
@@ -81,6 +60,7 @@ export class AppLayout {
 
     if (this.el.getAttribute('title')) {
       this.appLayoutTitle = this.el.getAttribute('title');
+      this.el.removeAttribute('title');
```
Comment (dani-cl-madrid) [inline src/custom-elements/src/components/app-layout/app-layout.tsx:63]: Please, add this action to the connect event as well, just in the case the component is reused.

**Review by dani-cl-madrid — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-185/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://476257edb88217611c5480e528eac71e1c09def9-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-185/2/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://93dc2bde642128a0e8bef1fa0d1838a2c37a7ad1-chi-pr.enterprise.ctl.io

**Review by dani-cl-madrid — APPROVED** [review]

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://484802ebbe544823d91cefe4f67c47ec895c8c89-chi-pr.enterprise.ctl.io

**Review by jllr — APPROVED** [review]

**Pull request merged**